code/src/components/Header.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import COLORS from './Colors'
+
+const Header = () => {
+  return (
+    <Container>
+      <Heading
+        type="text"
+        defaultValue="To do list" />
+    </Container>
+  )
+}
+
+const Container = styled.div`
+  background: ${COLORS.lightGrey};
+  height: 70px;
+  border-radius: 24px;
+  display: flex;
+  padding: 0 20px;
+  `
+
+const Heading = styled.input`
```

Comment:
Why you used input here? If you wanted to give your user the possibility to change the title you should create a button which would toggle between input/edit mode and read-only <h1-6>